### DISCUSSION TITLE
ZSE11 reports temperature in C even when set to report in F

### DISCUSSION THREAD

**Discussion by TheKorn2** [zwave-js/zwave-js#7283] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?

ZSE11 is reporting temperature in C instead of F.

What did you expect to happen instead?

Temperature should be reported in F.

1. Open ZSE11.
2. Open configuration
3. See configuration in F.  Change from to C, push, change back to F, push just for good measure.
4. Refresh multilevel sensor reading, see 170-49-0-air temperature still reported in C.

Anything else we should know?

Re-interviewed device, no change.  (Device is hard powered and was included that way, in case it matters.)

Seems to have started recently, but I had my system frozen for a good six months chasing down a problem I gave up diagnosing so that's not really helpful.

<img width="1179" alt="zse11" src="https://github.com/user-attachments/assets/b89d9305-337b-48e9-81e8-499e50aad694">

### Software versions

Driver (node-zwave-js): 13.9.0

Z-Wave JS UI: 9.23.0

Home Assistant Z-Wave Integration: none

Home Assistant Z-Wave JS Addon: none

### Z-Wave Controller (Stick/Dongle/...)

aeotec 5 usb stick

### Device information

Manufacturer: zooz
Model name: zse11
Node ID: 170

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwavejs_2024-10-15.log.gz](https://github.com/user-attachments/files/17377983/zwavejs_2024-10-15.log.gz)

**Comment by AlCalzone** [comment, marked as answer]
Thanks for reporting. Someone else also found the root cause: https://github.com/zwave-js/node-zwave-js/discussions/7283
Will be fixed in https://github.com/zwave-js/node-zwave-js/pull/7286 ASAP.
